packages/google_maps_flutter/example/pubspec.yaml

```diff
@@ -1,6 +1,5 @@
 name: google_maps_flutter_tizen_example
 description: Demonstrates how to use the google_maps_flutter_tizen plugin.
-publish_to: none
```

Comment:
Why did you remove this?
https://dart.dev/tools/pub/pubspec#publish_to

Reply:
Oh. It's my mistake.